src/template-files.ts

```diff
@@ -53,3 +54,84 @@ export const launchJson: {} = {
         }
     ]
 };
+
+const taskJsonForJava: {} = {
+    version: '2.0.0',
+    tasks: [
+        {
+            taskName: 'Launch Function App',
+            identifier: 'launchFunctionApp',
+            linux: {
+                command: 'sh -c "mvn clean package && func host start --script-root %path%"'
+            },
+            osx: {
+                command: 'sh -c "mvn clean package && func host start --script-root %path%"'
+            },
+            windows: {
+                command: 'powershell mvn clean package; func host start --script-root %path%'
+            },
+            type: 'shell',
+            isBackground: true,
+            presentation: {
+                reveal: 'always'
+            },
+            problemMatcher: [
+                {
+                    owner: 'azureFunctions',
+                    pattern: [
+                        {
+                            regexp: '\\b\\B',
+                            file: 1,
+                            location: 2,
+                            message: 3
+                        }
+                    ],
+                    background: {
+                        activeOnStart: true,
+                        beginsPattern: '^.*Stopping host.*',
+                        endsPattern: '^.*Job host started.*'
+                    }
+                }
+            ]
+        }
+    ]
+};
+
+const launchJsonForJava: {} = {
+    version: '0.2.0',
+    configurations: [
+        {
+            name: localize('azFunc.attachToFunc', 'Attach to Azure Functions'),
+            type: 'java',
+            request: 'attach',
+            hostName: 'localhost',
+            port: 5005,
+            preLaunchTask: taskId
+        }
+    ]
+};
+
+function stringifyJSON(data: {}): string {
+    return JSON.stringify(data, null, '    ');
+}
+
+export function getTasksJson(language: string, args: string): string {
```

Comment:
Just modify the object is good for me.

So shall we define the structure of the task json object here? Otherwise the object field cannot be accessed by ```.```. Or we use ```[]``` to access the field, but this may raise tslint error.

Reply:
Whatever works for you - I care more about making the output consistent than how you fix it

In addition, don't feel like you _have_ to follow every tslint rule. This might be a case where you just ignore the rule
<img width="667" alt="screen shot 2017-11-06 at 7 59 53 pm" src="https://user-images.githubusercontent.com/11282622/32476605-3c5e4a1e-c32d-11e7-966f-37be047329bd.png">

Reply:
Thanks for the suggestion.